pkg/test/framework/components/istio/kubealpha.go

```diff
@@ -0,0 +1,241 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package istio
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path"
+	"sync"
+
+	"github.com/hashicorp/go-multierror"
+
+	"istio.io/istio/pkg/test/deployment"
+	ienv "istio.io/istio/pkg/test/env"
+	"istio.io/istio/pkg/test/framework/components/environment/kube"
+	"istio.io/istio/pkg/test/framework/resource"
+	"istio.io/istio/pkg/test/scopes"
+)
+
+type installerComponent struct {
+	id          resource.ID
+	settings    Config
+	ctx         resource.Context
+	environment *kube.Environment
+}
+
+var _ io.Closer = &installerComponent{}
+var _ Instance = &installerComponent{}
+var _ resource.Dumper = &installerComponent{}
+
+// ID implements resource.Instance
+func (i *installerComponent) ID() resource.ID {
+	return i.id
+}
+
+func (i *installerComponent) Settings() Config {
+	return i.settings
+}
+
+func (i *installerComponent) Close() error {
+	if !i.settings.DeployIstio {
+		return nil
+	}
+	namespaces := GetDeployedNamespaces(i.settings)
+	errchan := make(chan error, len(namespaces))
+	wg := &sync.WaitGroup{}
```

Comment:
I think implicit zero values are more idiomatic (var wg sync.WaitGroup), though again, I'm not sure anybody cares in this codebase